unison-core/src/Unison/NamesWithHistory.hs

```diff
@@ -71,7 +71,6 @@ data Diff = Diff
     addedNames :: Names,
     removedNames :: Names
   }
-  deriving (Show)
```

Comment:
Just curious why removing this? I see you're still deriving `Show` for `Names` above.